src/helics/apps/MultiBroker.hpp

```diff
@@ -30,7 +30,8 @@ class MultiBroker: public CoreBroker {
     std::unique_ptr<CommsInterface>
         masterComm;  //!< the primary comms object or the one that links with the master
     NetworkBrokerData netInfo{
-        InterfaceTypes::TCP};  //!< structure containing the networking information
+        NetworkBrokerData::InterfaceTypes::TCP};  //!< structure containing the
```

Comment:
How were you deciding which parts use InterfaceTypes from NetworkBrokerData vs gmlc::networking?

Reply:
they are aliased to eachother so are the same, but what I sort of tried to do is if it was directly in relation to NetworkBrokerData I would use the networkBrokerData alias.    Might need to be a bit more consistent or just get rid of the NetworkBrokerData alias to make sure everything is consistent.